app/jobs/parse_report_job.rb

```diff
@@ -28,7 +28,7 @@ def save_all
     notify_payload_tracker(:processing, "Job #{jid} is now processing")
     parser.save_all
     notify_payload_tracker(:success, "Job #{jid} has completed successfully")
-  rescue ::EmptyMetadataError, ::WrongFormatError => e
+  rescue ::MissingIdError, ::WrongFormatError, ::InventoryHostNotFound => e
```

Comment:
This merely ensures the expected errors (missing ID, or missing host in the inventory) are logged in payload tracker and in our logs